DoorGate/DoorState.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  DoorState.swift
+//  DoorGate
+//
+//  Created by Yuliani Noriega on 1/26/18.
+//  Copyright © 2018 Yuliani Noriega. All rights reserved.
+//
+
+//Also known as the accumulator
+enum DoorState: Int {
+    case empty = 0, occupied, locked
```

Comment:
so occupied and locked are not represented by an int value in the enum  but empty is? 🤔 not sure I follow 100%. are occupied and locked automatically designated 1 and 2?

Reply:
commas mean iteration, its the same as doing 

case empty = 0,
case occupied = 1,
case locked = 2